### ISSUE TITLE
Nanospread: Selecting "Both" under "Show what on the Square" in Settings does not check the button.

### ISSUE THREAD

**Issue by natchapman** [StopSign/stopsign.github.io#12]
you need to add 

```javascript
    } else if (settings.selectShowNoneOrNanitesOrAmount === 3) {
        document.getElementById('selectShowBoth').checked = "checked";
```
to this.updateSettings() in view.js

**Comment by natchapman** [comment]
also you need to clear displayNum2colxrowy when you switch from 'Both' to something else.

**Comment by natchapman** [comment]
fixed by 3015f55380fb5ea73d025fecc49f7e429e63d8a1
